.github/workflows/manual-build.yml

```diff
@@ -9,20 +9,31 @@ env:
   FASTLANE_XCODEBUILD_SETTINGS_RETRIES: 6 # increase timeout for xcodebuild settings. We have seen this command take a bit longer to do on CI server. 
 
 jobs:
-  deploy-app:
-    name: Re-release the latest stable build 
-    runs-on: macos-12
+  get-latest-git-tag:
+    name: Get latest git tag
+    runs-on: ubuntu-latest
     steps:
-      - name: Get latest git tag 
+      - name: Get latest git tag
         uses: octokit/request-action@v2.1.4
         id: get_latest_release
         with:
-          route: GET /repos/customerio/amiapp-reactnative/releases/latest
+          route: GET /repos/customerio/amiapp-flutter/releases/latest
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}                    
-      - run: echo "LATEST_GIT_TAG=${{ fromJson(steps.get_latest_release.outputs.data).tag_name }}" >> $GITHUB_ENV      
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # Checkout the git tag right before building since older git tags might not have the necessary files
+      # that the newer CI config asks for. Do all CI environment prep before checking out the git tag.
+      - run: echo "LATEST_GIT_TAG=${{ fromJson(steps.get_latest_release.outputs.data).tag_name }}" >> $GITHUB_ENV
+      - name: Checkout latest git tag
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.LATEST_GIT_TAG }}
+          clean: false # to not delete the files created in environment setup step
 
-      - uses: actions/checkout@v2 # checkout to be able to run CI config files below 
+  deploy-android-app:
+    runs-on: macos-12
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
Responded in https://github.com/customerio/amiapp-flutter/pull/6#discussion_r1122104861